doc/release/release_0.16.rst

```diff
@@ -0,0 +1,386 @@
+Announcement: scikit-image 0.16.0
+=================================
+
+We're happy to announce the release of scikit-image v0.16.0!
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+For more information, examples, and documentation, please visit our website:
+
+https://scikit-image.org
+
+Starting from this release, scikit-image will follow the spirit of the recently
+introduced numpy deprecation policy -- NEP 29
+(https://github.com/numpy/numpy/blob/master/doc/neps/nep-0029-deprecation_policy.rst). 
+Accordingly, scikit-image 0.16 drops the support for Python 3.5.
+This release of scikit-image officially supports Python 3.6 and 3.7.
```

Comment:
@stefanv mentioned on Zulip that maybe we should release with 3.8 next week. But imho we can just make a 0.16.1 release for that.

Reply:
Are our core dependencies built for 3.8? We can likely even upload without matplotlib. Numpy has wheels so we can build for 3.8 without tests today for sure.

Reply:
The current release of `PyWavelets` (actually releases `1.0.0` through `1.0.3`) is broken on Python 3.8 due to a deprecated import from `collections`. This has already been fixed for a while in `PyWavelets` master and I plan to release a v1.1.0 within the week that will include the fix. 

Also, PyWavelets 0.4.x and 0.5.x may work with 3.8, but I haven't tested to verify this (the problematic import statement had not yet been introduced at that time)

Reply:
Hmm good to know. I guess we would need to bump minimum version requirements in some cases. 

I think an untested scikit-image 3.8 (build the wheel with numpy, but don't test it) might be ok though since we have been quite good at catching warnings

Reply:
Not sure what you mean by build the wheel but don't test it; it's very likely to be broken if it's not tested.

Reply:
I think @jni's suggestion is good. If/when the tests pass on 3.8, release a micro shouldn't be hard and fast. So unless the release takes sufficiently long that 3.8 is out, I think we should go ahead and release without 3.8 wheels.

Reply:
+1 on releasing without 3.8 wheels initially. Those can always be added later and shouldn't even require tagging a new release unless there are any actual 3.8-related bugs that crop up.